charts/karmada/templates/karmada-static-resource-job.yaml

```diff
@@ -11,6 +11,9 @@ metadata:
 spec:
   parallelism: 1
   completions: 1
+  {{- if semverCompare ">=1.23.0-0" .Capabilities.KubeVersion.GitVersion }}
```

Comment:
Where does the `.Capabilities.KubeVersion.GitVersion` come from?

Is this version always comparable? I mean I can't assume all Kubernetes version format is `x.x.y-0xxx`.

Reply:
> Where does the `.Capabilities.KubeVersion.GitVersion` come from?

It's helm's [built-in objects](https://helm.sh/docs/chart_template_guide/builtin_objects/).

> Is this version always comparable? I mean I can't assume all Kubernetes version format is `x.x.y-0xxx`.

Yes, don't worry about the Kubernetes version format, it is semantic versioning. So the `-0` suffix here is used to find pre-release versions, e.g., 1.24-beta.1 will still be considered greater than 1.23, see https://masterminds.github.io/sprig/semver.html for more information.